scipy/ndimage/tests/test_measurements.py

```diff
@@ -1084,3 +1098,214 @@ def test_stat_funcs_2d():
     max = ndimage.maximum(a, labels=lbl, index=[1, 2])
     assert_array_equal(max, [9, 5])
 
+
+class TestWatershedIft:
```

Comment:
this entire test class snuck in with the backport, but really just the last test `test_watershed_ift08` was explicitly added in a backport PR; the more conservative test merge I did this evening just split that test off, but not sure if other tests are "harmful"